ansible/tasks/postgres-extensions/24-pgroonga.yml

```diff
@@ -1,4 +1,17 @@
 # groonga and pgroonga
+- name: groonga - download & install dependencies
+  apt:
+    pkg:
+      - groonga-normalizer-mysql
```

Comment:
Oh, wait. Can we use `groonga-normalizer-package` in this environment? (`postgres:15.1` that is based on Debian GNU/Linux bullseye?)

How about using deb packages for Groonga/PGroonga provided by https://packages.groonga.org/ instead of building Groonga/PGroonga manually? PGroonga's Docker images use them: https://github.com/pgroonga/docker/blob/master/debian/15/Dockerfile

See also: https://pgroonga.github.io/install/debian.html

Reply:
Unfortunately the images we build and distribute also bundle extensions that are compiled during the build process, as part of keeping the software bill of materials in check so we can have dependency versions locked down - thus avoiding vulnerabilities introduced upstream or dependency chains breaking, additionally maintaining compliance since project databases might contain sensitive information.

The MeCab tokenizer is already present in Groonga's source code and easily to install during compilation, albeit `groonga-normalizer-mysql` isn't 🤔 and we might have to also build it from source.

Reply:
I see.

I think that we can remove `groonga-normalizer-mysql`. It's for using MySQL compatible normalization (collation) in Groonga. In general, PGroonga users don't use it.

Reply:
I see. Thank you for the information and all the feedback!

I shall be merging this to make Docker images available - we're aiming to roll this out for new projects on the Supabase platform sometime next week.

Reply:
@kou support for this rolled out today for new projects, and projects which pause and then unpause themselves to update to the latest version.